### ISSUE TITLE
Uncaught Invariant Violation: addComponentAsRefTo(...)

### ISSUE THREAD

**Issue by jephir** [auth0/lock-passwordless#114]
I'm trying to use `lock-passwordless` in my React application. After signing in using email code and seeing "Thanks for signing in.", this error appears in console:

```
Uncaught Invariant Violation: addComponentAsRefTo(...): Only a ReactOwner can have refs. You might be adding a ref to a component that was not created inside a component's `render` method, or you have multiple copies of React loaded (details: https://fb.me/react-refs-must-have-owner).
```

My application is using `react@15.1.0`. NPM also shows this error: `UNMET PEER DEPENDENCY react@0.14.8` under the `lock-passwordless` tree.

**Comment by gnandretta** [comment]
Lock passwordless [works with React 0.14](https://github.com/auth0/lock-passwordless/blob/df183e846e76b45ce874f87eb6721ca575da21f0/package.json#L71). I'll schedule some time to perform the upgrade to React 15.

**Comment by nlindroos** [comment]
@gnandretta Any updates on a schedule for React 15 support?

**Comment by ChristianDavis** [comment]
I get this as well, but on initial launch of the lock

**Comment by luisrudge** [comment]
I'm having the same issue:
![image](https://cloud.githubusercontent.com/assets/941075/19930379/5c6249ca-a0ef-11e6-94ff-11827f95322f.png)

**Comment by nlindroos** [comment]
@luisrudge You need to use React v0.14 for now, since development on this library apparently has stopped.

**Comment by ChristianDavis** [comment]
We were able to get passed this by externalizing react from the project through webpack

**Comment by luisrudge** [comment]
@ChristianDavis right.. but this won't work with something like nextjs (https://github.com/zeit/next.js)

**Comment by djmitche** [comment]
I think you need to list React as a peer dependency.  Loading two different versions of React is always going to lead to this issue.  The other option would be to release a different version of auth0-lock-passwordless for each version of React, and asking users to select the lock version matching their react version.

**Comment by jeffgorder** [comment]
I'm getting this same issue.
